platform/features/plot/src/elements/PlotTickGenerator.js

```diff
@@ -57,12 +57,19 @@ define(
         PlotTickGenerator.prototype.generateTicks = function (start, span, count, format) {
             var step = span / (count - 1),
                 result = [],
+                tickVal = '',
                 i;
 
             for (i = 0; i < count; i += 1) {
+                tickVal = format(i * step + start);
+                if (tickVal !== undefined) {
+                    // Make the tick value have its ellipsis on the least significant left side by reversing it here,
+                    // and then reversing it again via CSS.
+                    tickVal = tickVal.toString().split('').reverse().join('');
+                }
```

Comment:
Understand why this is done, but it's a little funny for JS here to be so concerned about what CSS is going to do. The more Angular approach to this would be a `filter` (which we don't integrate with, but should be able to add trivially). Will try this out and submit a PR to this branch